### ISSUE TITLE
Installation fails with: SyntaxWarning: "is not" with a literal.

### ISSUE THREAD

**Issue by ruedigerkupper** [leamas/ddupdate#40]
Trying to install the ddupdate package from Ubuntu focal repo fails with:
```
/usr/lib/python3/dist-packages/ddupdate/ddplugin.py:178: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  words = [ word for word in line.split(' ') if word is not '' ]
```
and service ist not istalled:
```
systemctl status ddupdate.timer 
Unit ddupdate.timer could not be found.
```

**Comment by ruedigerkupper** [comment]
ddupdate-config fails with
```
Starting service and displaying logs
Failed to start ddupdate.service: Unit ddupdate.service has a bad unit file setting.
See user logs and 'systemctl --user status ddupdate.service' for details.
No journal files were found.
-- No entries --
Use "journalctl --user -u ddupdate.service" to display logs.
Shall I run service regularly (Yes/No) [No]: Yes

Starting and enabling ddupdate.timer
Job failed. See "journalctl -xe" for details.
```

**Comment by leamas** [comment]
Ack, needs a fix. It's new behaviour in recent python. Will look into it, time permitting.

**Comment by leamas** [comment]
The root cause here seems to be a misconfigured /usr/lib/python3/dist-packages/ddupdate/install.conf which reads (excerpt)
```
root = debian/tmp
install_purelib = debian/tmp/usr/lib/python3/dist-packages
install_platlib = debian/tmp/usr/lib/python3/dist-packages
install_lib = debian/tmp/usr/lib/python3/dist-packages/
install_headers = debian/tmp/usr/include/python3.7/ddupdate
install_scripts = debian/tmp/usr/bin
install_data = debian/tmp/usr

```
You should be able to walk around the problem by fixing this file. Note that the syntax warning needs to be fixd as well, but it should actually not be a blocking error.

**Comment by ruedigerkupper** [comment]
Thank you, I already wondered, as it is a warning only and should not actually break execution.
Any idea what is misconfigured in install.conf?

**Comment by leamas** [comment]
This is a debian packaging; it will take some time before I can submit a new build. Leaving bug open.

**Comment by ruedigerkupper** [comment]
Separated the two issues.

**Comment by ruedigerkupper** [comment]
Oops, you were faster ;-).

**Comment by leamas** [comment]
The warning is already fixed in master: 7b7f86db75

**Comment by ruedigerkupper** [comment]
So it actually was so easy, thanks! I leave it to you which of #40 and #41 to keep and which to close, sorry for the mess :-).

**Comment by leamas** [comment]
I'm closing this as fixed in 7b7f86d. 

Thanks for reporting!
